roles/apt/templates/30apt-proxy.j2

```diff
@@ -1,2 +1,2 @@
 // {{ ansible_managed }}
-Acquire::http::Proxy "{{ apt_proxy }}";
+Acquire::http::Proxy "http://{{ inventory_hostname }}:8000/";
```

Comment:
Ohh... right.  Hmm....
I am used to all these files living on the pxe server as static files.
I was thinking ansible creates the static file on the pxe server and then the other machines pull a copy. 
but that isn't how things work here.

this bit is 
 31 # d-i puts the apt config in apt.conf, but we prefer a .d file

d-i puts the correct value in apt.conf, how about we just leave it there?

or patch d-i :p